src/main/java/frc/robot/subsystems/pivot/PivotIO.java

```diff
@@ -32,12 +37,32 @@ public default boolean isPivotWithinAcceptableError() {
     return false;
   }
 
+  /**
+   * Sets the output of the pivot
+   *
+   * @param output output value from -1.0 to 1.0
+   */
   public default void setPivotSpeed(double output) {}
 
+  /**
+   * Sets the voltage of the pivot motors
+   *
+   * @param volts the voltage
+   */
   public default void setVoltage(double volts) {}
 
+  /**
+   * sets the pivot angle(rotations) shooter
+   *
+   * @param angle the desired angle in rotations
+   */
   public default void setPivotAngle(double angle) {}
 
+  /**
+   * Gets the target angle of the pivot in degrees
```

Comment:
Are these units correct?

Reply:
im just gonna do everything in pivotIO as rotations